components/resources/resources-core/src/main/resources/META-INF/dirigible/resources-core/services/menus/help.js

```diff
@@ -43,6 +43,13 @@ const defaultMenuItems = [
 		action: "open",
 		order: 5,
 		data: "http://download.dirigible.io/",
+		divider: false
+	},
+	{
+		label: "Spring Boot Admin",
```

Comment:
Maybe `Admin Panel` instead of `Spring Boot Admin`?

Reply:
as you wish